src/composables/multisig.ts

```diff
@@ -0,0 +1,85 @@
+import { ref } from '@vue/composition-api';
+import type { Store } from 'vuex';
+import multisigContract from '@aeternity/ga-multisig-contract/SimpleGAMultiSig.aes';
+import { Crypto } from '@aeternity/aepp-sdk';
+import { MultisigAccount, MultisigCreationStep, MultisigCreationSteps } from '../types';
+import { useSdk } from './sdk';
+
+interface UseMultisigOptions {
+  /**
+   * TODO: Temporary solution to avoid dependency circle
+   */
+  store: Store<any>
+}
+
+const MINIMUM_REQUIRED_SIGNERS = 2;
+
+/**
+ * This composable provides methods related to multisig operations
+ */
+export function useMultisig({ store }: UseMultisigOptions) {
+  const { getDrySdk, getSdk } = useSdk({ store });
+
+  const multisigProgress = ref<MultisigCreationSteps>(MultisigCreationStep.preparing);
+
+  /**
+   * @param {number} noOfConfirmations no of minimum confirmation needed to process a transaction
+   * @param {string[]} signers
+   */
+  async function deployMultisigAccount(
+    noOfConfirmations: number,
+    signers: string[],
+  ): Promise<MultisigAccount> {
+    if (signers.length < MINIMUM_REQUIRED_SIGNERS) throw Error(`Minimum ${MINIMUM_REQUIRED_SIGNERS} signers required`);
+    if (noOfConfirmations > signers.length) throw Error('Number of confirmations exceed amount of signers');
```

Comment:
what do you think about moved error message to translation or another file?

Reply:
This is for the developers using this method. Hence I guess it could be kept here.